### ISSUE TITLE
PoB Portable periodically crashing

### ISSUE THREAD

**Issue by goldmango** [PathOfBuildingCommunity/PathOfBuilding#5909]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Screenshots of crash/graphical issue, version and options menu

Data from Event Viewer:
Faulting application name: Path of Building.exe, version: 0.0.0.0, time stamp: 0x606d1b4c
Faulting module name: igxelpicd32.dll, version: 31.0.101.3962, time stamp: 0x6397b70f
Exception code: 0xc0000005
Fault offset: 0x0028e4c5
Faulting process id: 0x0x43DC
Faulting application start time: 0x0x1D9663992191719
Faulting application path: C:\PathOfBuildingCommunity-Portable-2.23.0\Path of Building.exe
Faulting module path: C:\WINDOWS\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_a20fccc4824e3254\igxelpicd32.dll
Report Id: 5dbec313-e5d1-45d5-b8bf-1430cac33c8d
Faulting package full name: 
Faulting package-relative application ID: 

### What are your system specifications and configuration?

System	Windows 11 Home (latest update)
Processor	12th Gen Intel(R) Core(TM) i9-12900H   2.50 GHz
RAM	32.0 GB
System type	64-bit operating system, x64-based processor
Graphic card	Nvidia GeForce RTX 3070 Ti

### How to reproduce the issue

Actually do not know how to reproduce.
Usually it happens when PoB was not active, was in tray. It happens unpredictable. Few times it was while working with builds. Importing some of them, change their settings, etc.
After crush happened - display become black, then after 3-4 seconds display show again and it show that a lot of application closed (poe, awa poe trade, sometimes chrome) and PoB application is not responding.
That happens before also (during league 3.20 start)

**Comment by goldmango** [comment]
@QuickStick123 same on the PoB (not Portable), same system.

Log from Event Viewer:

```
Faulting application name: Path of Building.exe, version: 0.0.0.0, time stamp: 0x606d1b4c
Faulting module name: igxelpicd32.dll, version: 31.0.101.3962, time stamp: 0x6397b70f
Exception code: 0xc0000005
Fault offset: 0x0076b35b
Faulting process id: 0x0x2E70
Faulting application start time: 0x0x1D96AC2E02A4777
Faulting application path: C:\Path of Building Community\Path of Building.exe
Faulting module path: C:\WINDOWS\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_a20fccc4824e3254\igxelpicd32.dll
Report Id: 429a1165-ad6d-4cdf-8f31-a5dcd85a6640
Faulting package full name: 
Faulting package-relative application ID: 
```

```
Faulting application name: Path of Building.exe, version: 0.0.0.0, time stamp: 0x606d1b4c
Faulting module name: igxelpicd32.dll, version: 31.0.101.3962, time stamp: 0x6397b70f
Exception code: 0xc0000005
Fault offset: 0x0028e4c5
Faulting process id: 0x0x5F68
Faulting application start time: 0x0x1D96ABFE1B47761
Faulting application path: C:\Path of Building Community\Path of Building.exe
Faulting module path: C:\WINDOWS\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_a20fccc4824e3254\igxelpicd32.dll
Report Id: 891b511f-9f85-45b6-a75a-e851fcce0930
Faulting package full name: 
Faulting package-relative application ID: 
```

```
Faulting application name: Path of Building.exe, version: 0.0.0.0, time stamp: 0x606d1b4c
Faulting module name: igxelpicd32.dll, version: 31.0.101.3962, time stamp: 0x6397b70f
Exception code: 0xc0000005
Fault offset: 0x0028e4c5
Faulting process id: 0x0x4A70
Faulting application start time: 0x0x1D969E16D46AD45
Faulting application path: C:\Path of Building Community\Path of Building.exe
Faulting module path: C:\WINDOWS\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_a20fccc4824e3254\igxelpicd32.dll
Report Id: 170c0fe1-7428-46e4-9aba-246753aebe29
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by QuickStick123** [comment]
Try reinstalling graphics drivers and turn off any overclock or other system instability that could be causing it.
You can try the angle branch (https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/4736#issuecomment-1211478087) but it sounds like a wider issue not entirely limited to PoB.

**Comment by Wires77** [comment]
Probably fixed by #7399
